hammerspoon/super.lua

```diff
@@ -126,7 +126,7 @@ superDuperModeNavListener = eventtap.new({ eventTypes.keyDown }, function(event)
     l = 'right',
   }
 
-  local keystroke = charactersToKeystrokes[event:getCharacters(true)]
+  local keystroke = charactersToKeystrokes[event:getCharacters(true):lower()]
```

Comment:
Nice fix. :bow:

---

xref: [Hammerspoon API docs](http://www.hammerspoon.org/docs/hs.eventtap.event.html#getCharacters) for anyone following along:

> `hs.eventtap.event:getCharacters([clean]) -> string or nil`
>
> ...
>
> If `clean` is true, all modifiers except for Shift are stripped from the character before converting to the Unicode character represented by the keypress.